asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -123,6 +123,15 @@ The Confluence Publisher is configured with the help of a Maven plugin. A typica
 | The password of the user to use for publishing.
 | mandatory
 
+| publishingStrategy
+a| The strategy to use when publishing to confluence. Possible values:
```

Comment:
this `a|` is wrong right? `a` should be removed.

Reply:
This is required in order to force AsciiDoc to render cell contents in formatted way (i.e. the bullet list).